diff -u original-src/lvm.c src/lvm.c
--- original-src/lvm.c	2020-04-10 18:08:01.093752204 -0600
+++ src/lvm.c	2020-04-10 18:05:25.360765887 -0600
@@ -15,6 +15,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <unistd.h>
 
 #include "lua.h"
 
@@ -30,6 +31,7 @@
 #include "ltm.h"
 #include "lvm.h"
 
+#define LUA_USE_JUMPTABLE 0
 
 /*
 ** By default, use jump tables in the main interpreter loop on gcc
@@ -1079,6 +1081,8 @@
     Instruction i;  /* instruction being executed */
     StkId ra;  /* instruction's A register */
     vmfetch();
+    fprintf(stderr, "%d\n", GET_OPCODE(i));
+    usleep(100000);
     lua_assert(base == ci->func + 1);
     lua_assert(base <= L->top && L->top < L->stack + L->stacksize);
     /* invalidate top for instructions not expecting it */
